docs/_data/table/en/user-configuration.yml

```diff
@@ -85,6 +85,13 @@ alluxio.user.network.netty.channel.pool.gc.threshold.ms:
   A netty channel is closed if it has been idle for more than this threshold.
 alluxio.user.network.netty.channel.pool.disabled:
   Disable netty channel pool. This should be turned on if the client version is >= 1.3.0 but server version is <= 1.2.x.
+alluxio.user.ufs.block.read.location.policy:
+  The policy for block workers to read the UFS blocks.
```

Comment:
Minor nit: consider "The policy block workers follow for reading UFS blocks.", instead. I don't fully understand the context though.